Update chapter3.adoc (#28)
riscv-non-isa/riscv-security-model: specification/src/chapter3.adoc

==== Capability based architecture

> * CHERI - a mature technique that uses capability-based memory protection for spatial and temporal memory safety, highly efficient compartmentalisation, and control-flow robustness.  Source code has to be recompiled to capture memory safety properties inherent in the source language.

Reviewer: ```suggestion
* CHERI - an ISA technique that uses capability-based memory protection for spatial and temporal memory safety, compartmentalization, and control-flow enforcement.  Source code has to be recompiled to capture memory safety properties inherent in the source language.
```
